tools/packaging/scripts/lib.sh

```diff
@@ -119,5 +119,8 @@ get_config_version() {
 get_last_modification() {
 	local file="${1}"
 
-	echo "$(git log -1 --pretty=format:"%H" ${file})"
+	dirty=""
+	[ $(git status --porcelain | grep modified | grep "${file}") ] && dirty="-dirty"
```

Comment:
With git 2.34.3 on Fedora 34 `git status --porcelain` returns the modified lines with prefix `M` instead of `modified`.